src/gas-price/gas-price.ts

```diff
@@ -47,6 +47,7 @@ export const clearExpiredStoreGasPrices = (chainId: string, providerName: string
  * @param providerName
  * @param sponsorWalletAddress
  */
+// TODO: Why is this unused?
```

Comment:
I don't think the `update-transactions` file existed and we probably forgot to use it 😄

This function should be used directly after submitting an update transaction, but only if it's the first time submitting a particular update (so not reset on when scaling is applied)..